src/Tribe/Commerce/PayPal/Orders/Table.php

```diff
@@ -179,7 +179,9 @@ public function column_order( $item ) {
 	 * @return string
 	 */
 	public function column_total( $item ) {
-		return tribe_format_currency( number_format( $item['line_total'], 2 ) );
+		$post_id = Tribe__Utils__Array::get( $_GET, 'post_id', null );
```

Comment:
why not `tribe_get_request_var()`?

Reply:
to skip a check on the `$_POST` array; it seems unlikely there would ever be the need to look up `$_POST` when rendering the table